server/modules/user/manager/userManager.js

```diff
@@ -90,12 +93,17 @@ const updateUser = _userFetcher(_updateUser)
 // ==== DELETE
 
 const deleteUser = async (user, surveyId, userUuidToRemove, client = db) =>
-  await Promise.all([
-    AuthGroupRepository.deleteUserGroup(surveyId, userUuidToRemove, client),
-    ActivityLogRepository.insert(
-      user, surveyId, ActivityLog.type.userRemove, { uuid: userUuidToRemove }, false, client
-    )
-  ])
+  await client.tx(async t => {
+    const userToRemove = await UserRepository.fetchUserByUuid(userUuidToRemove, t)
+    const logContent = {
+      [ActivityLog.keysContent.uuid]: userUuidToRemove,
+      [ActivityLog.keysContent.userName]: User.getName(userToRemove)
+    }
+    return await Promise.all([
+      AuthGroupRepository.deleteUserGroup(surveyId, userUuidToRemove, t),
+      ActivityLogRepository.insert(user, surveyId, ActivityLog.type.userRemove, logContent, false, t)
+    ])
+  })
```

Comment:
Users are never deleted from database. In the log I would save the userUuid to remove only, and in the fetch I would get the username

Reply:
ok